shared/spotify-api/src/types.ts

```diff
@@ -185,6 +186,7 @@ export namespace Spotify {
 
     export interface Episode {
       audio_preview_url: string | null;
+      content_type: 'PODCAST_EPISODE';
```

Comment:
Is this an example where the fields are available in the actual request, but aren't documented? I don't see these listed in the [documentation](https://developer.spotify.com/documentation/web-api/reference/get-an-episode) (this was my biggest pain point developing this the first time). I based it off of [this episode](https://open.spotify.com/episode/7aGd7sVE4Budo9AHkr7QQC) as my test, so perhaps the Apollo episodes have more data available?

Reply:
Same as policies, yeah, this field appeared and TypeScript demanded I do something about it 😅

Reply:
Odd. I just ran one of the IDs added to the mocks against the Spotify API (`/episodes/1Wq3HgaoISjc9ZUmXdbaSK`) and got this:

```js
{
  audio_preview_url: 'https://podz-content.spotifycdn.com/audio/clips/7B95PnkRlqf3DH8mRn6be9/clip_281015_341015.mp3',
  description: 'Lenz Weber-Tronic and Jeff Auriemma discuss Apollo Client, React Server Components, and more! Originally streamed live in the Apollo Discord on 2023-05-30.  Links:   Full Transcript  Apollo Client   React Server Components   Apollo Client Next.js  Topics:    Caching in Apollo Client   Cache Metrics   React Server Components   Apollo Client 3.8   How to contact us   Apollo Client and React Native ',
  duration_ms: 3202844,
  explicit: false,
  external_urls: {
    spotify: 'https://open.spotify.com/episode/1Wq3HgaoISjc9ZUmXdbaSK'
  },
  href: 'https://api.spotify.com/v1/episodes/1Wq3HgaoISjc9ZUmXdbaSK',
  html_description: '<p>Lenz Weber-Tronic and Jeff Auriemma discuss Apollo Client, React Server Components, and more! Originally streamed live in the <a href="https://discord.gg/M6FT5cNR5B" rel="nofollow">Apollo Discord</a> on 2023-05-30.<br/></p><br/><p>Links:</p><br/><ul><li><a href="https://www.apollographql.com/blog/apollo-client/office-hours-apollo-client/" rel="nofollow">Full Transcript</a></li><li><a href="https://www.apollographql.com/docs/react/" rel="nofollow">Apollo Client</a></li><li><a href="https://legacy.reactjs.org/blog/2020/12/21/data-fetching-with-react-server-components.html" rel="nofollow">React Server Components</a></li><li><a href="https://github.com/apollographql/apollo-client-nextjs" rel="nofollow">Apollo Client Next.js</a></li></ul><br/><p>Topics:</p><br/><ul><li>Caching in Apollo Client</li><li>Cache Metrics</li><li>React Server Components</li><li>Apollo Client 3.8</li><li>How to contact us</li><li>Apollo Client and React Native</li></ul>',
  id: '1Wq3HgaoISjc9ZUmXdbaSK',
  images: [
    {
      height: 640,
      url: 'https://i.scdn.co/image/ab6765630000ba8a6d2087f0952ded824410aca0',
      width: 640
    },
    {
      height: 300,
      url: 'https://i.scdn.co/image/ab67656300005f1f6d2087f0952ded824410aca0',
      width: 300
    },
    {
      height: 64,
      url: 'https://i.scdn.co/image/ab6765630000f68d6d2087f0952ded824410aca0',
      width: 64
    }
  ],
  is_externally_hosted: false,
  is_playable: true,
  language: 'en',
  languages: [ 'en' ],
  name: 'Web Client Office Hours with Lenz Weber-Tronic',
  release_date: '2023-05-30',
  release_date_precision: 'day',
  resume_point: { fully_played: false, resume_position_ms: 0 },
  show: {
    available_markets: [
      'AD', 'AE', 'AG', 'AL', 'AM', 'AR', 'AT', 'AU', 'BA',
      'BB', 'BE', 'BF', 'BG', 'BH', 'BJ', 'BO', 'BR', 'BS',
      'BT', 'BW', 'BZ', 'CA', 'CH', 'CL', 'CO', 'CR', 'CV',
      'CW', 'CY', 'CZ', 'DE', 'DK', 'DM', 'DO', 'DZ', 'EC',
      'EE', 'EG', 'ES', 'FI', 'FJ', 'FM', 'FR', 'GB', 'GD',
      'GE', 'GH', 'GM', 'GR', 'GT', 'GW', 'GY', 'HK', 'HN',
      'HR', 'HT', 'HU', 'ID', 'IE', 'IL', 'IN', 'IS', 'IT',
      'JM', 'JO', 'JP', 'KE', 'KI', 'KN', 'KR', 'KW', 'LB',
      'LC', 'LI', 'LR', 'LS', 'LT', 'LU', 'LV', 'MA', 'MC',
      'ME', 'MG', 'MH', 'MK', 'ML', 'MT', 'MU', 'MV', 'MW',
      'MX', 'MY', 'MZ', 'NA', 'NE', 'NG', 'NI', 'NL', 'NO',
      'NR', 'NZ', 'OM', 'PA', 'PE', 'PG', 'PH', 'PL', 'PS',
      'PT', 'PW', 'PY', 'QA', 'RO', 'RS', 'SA', 'SB', 'SC',
      'SE', 'SG', 'SI', 'SK', 'SL', 'SM', 'SN', 'SR', 'ST',
      'SV', 'TH', 'TL', 'TN', 'TO', 'TR', 'TT', 'TV', 'TW',
      'UA', 'US', 'UY', 'VC', 'VN', 'VU', 'WS', 'XK', 'ZA'
    ],
    copyrights: [],
    description: "Apollonauts discuss GraphQL APIs, both what we're building and what we see in the community.",
    explicit: false,
    external_urls: {
      spotify: 'https://open.spotify.com/show/22ZdnFXLzdXDEmv8vpyPyD'
    },
    href: 'https://api.spotify.com/v1/shows/22ZdnFXLzdXDEmv8vpyPyD',
    html_description: 'Apollonauts discuss GraphQL APIs, both what we&#39;re building and what we see in the community.',
    id: '22ZdnFXLzdXDEmv8vpyPyD',
    images: [
      {
        height: 640,
        url: 'https://i.scdn.co/image/ab6765630000ba8a6d2087f0952ded824410aca0',
        width: 640
      },
      {
        height: 300,
        url: 'https://i.scdn.co/image/ab67656300005f1f6d2087f0952ded824410aca0',
        width: 300
      },
      {
        height: 64,
        url: 'https://i.scdn.co/image/ab6765630000f68d6d2087f0952ded824410aca0',
        width: 64
      }
    ],
    is_externally_hosted: false,
    languages: [ 'en' ],
    media_type: 'audio',
    name: 'Apollo GraphQL',
    publisher: 'Apollo GraphQL',
    total_episodes: 6,
    type: 'show',
    uri: 'spotify:show:22ZdnFXLzdXDEmv8vpyPyD'
  },
  type: 'episode',
  uri: 'spotify:episode:1Wq3HgaoISjc9ZUmXdbaSK'
}
```

I'd prefer to remove these extra fields since I'm not seeing them anywhere in the API.